lib/rhc/commands/app.rb

```diff
@@ -425,7 +425,8 @@ def show(app_name)
         case options.gears
         when 'quota'
           opts = {:as => :gear, :split_cells_on => /\s*\t/, :header => ['Gear', 'Cartridges', 'Used', 'Limit'], :align => [nil, nil, :right, :right]}
-          table_from_gears('echo "$(du --block-size=1 -s 2>/dev/null | cut -f 1)"', groups, opts) do |gear, data, group|
+          # quota 'blocks' are actually kb, so have to multiply by 1024, see 'man quota'
+          table_from_gears('echo "$(quota | egrep -v \'quotas|Filesystem\' | awk \'{ print $2 * 1024 }\')"', groups, opts) do |gear, data, group|
```

Comment:
@Miciah Actually, looking at the man page for quota:
> quota  displays  users' disk usage and limits.  By default only the user quotas are printed. **By default space usage and limits are shown in kbytes (and are named blocks  for  historical  reasons).**

So it actually is always 1024 (kbytes) not BLOCK_SIZE from mount.h  so it seems like hard-coding makes sense?